options/locale/locale_uk-UA.ini

```diff
@@ -1171,7 +1171,7 @@ issues.action_milestone_no_select=Етап відсутній
 issues.action_assignee=Виконавець
 issues.action_assignee_no_select=Немає виконавеця
 issues.opened_by=%[1]s відкрито <a href="%[2]s">%[3]s</a>
-pulls.merged_by=до <a href="%[2]s">%[3]</a> злито %[1]s
+pulls.merged_by=до <a href="%[2]s">%[3]s</a> злито %[1]s
```

Comment:
Does not seem translated on Crowdin…

Reply:
ah yes ... since it to reworded ... (english source)